packages/dashboard/src/integration/iot-dashboard-context-menu.spec.component.ts

```diff
@@ -0,0 +1,93 @@
+import { renderDashboard } from '../testing/renderDashboard';
+import { Widget } from '../types';
+
+const createWidget = (partialWidget?: Partial<Widget>): Widget => ({
+  x: 1,
+  y: 1,
+  z: 1,
+  width: 4,
+  height: 4,
+  componentTag: 'line-chart',
+  queries: [],
+  id: Math.random().toString() + new Date().toISOString(),
+  ...partialWidget,
+});
+
+it('renders a context menu on right click', () => {
+  renderDashboard({
+    dashboardConfiguration: {
+      viewport: { duration: '5m' },
+      widgets: [createWidget()],
+    },
+  });
+
+  // Select the grid area of the dashboard.
+  cy.get('iot-dashboard-internal').children().first().rightclick();
+
+  cy.get('iot-context-menu').should('be.visible');
```

Comment:
i think were going to want at least one screenshot of it to catch weird style/build regressions

Reply:
👍 great point. added.